src/nvidia_rag/rag_server/server.py

```
@@ -666,6 +722,78 @@ def metrics_endpoint():
+                reranker_model=CONFIG.ranking.model_name.strip('"'),
+                vlm_model=CONFIG.vlm.model_name.strip('"'),
+            ),
+            endpoints=EndpointsDefaults(
+                llm_endpoint=CONFIG.llm.server_url.strip('"'),
+                embedding_endpoint=CONFIG.embeddings.server_url.strip('"'),
+                reranker_endpoint=CONFIG.ranking.server_url.strip('"'),
+                vlm_endpoint=CONFIG.vlm.server_url.strip('"'),
+                vdb_endpoint=CONFIG.vector_store.url,
+            ),
+        )
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)